src/context/Providers.tsx

```diff
@@ -25,7 +26,9 @@ export default async function Providers({
           disableTransitionOnChange
         >
           <WagmiProvider>
-            <EASClientProvider>{children}</EASClientProvider>
+            <EASClientProvider>
+              <AuthProvider>{children}</AuthProvider>
+            </EASClientProvider>
```

Comment:
AuthProvider should wrap EASClientProvider right?
- EAS might use Auth 👍 
- when will Auth ever need EAS? 🤔